pkg/lib/cockpit.d.ts

```diff
@@ -98,6 +99,8 @@ declare module 'cockpit' {
         changed(): void;
     }
 
+    function event_target<T, EM extends EventMap>(obj: T): T & EventSource<EM>;
```

Comment:
Oh, thanks! That was the magic syntax which I missed when trying to ts-ify pkg/apps/appstream.js in #21334, so I didn't add that commit (nice followup for me)